testing/browser-functional/browser-echo-bot/package.json

```diff
@@ -35,5 +35,8 @@
     "webpack": "^4.35.3",
     "webpack-cli": "^3.3.6",
     "webpack-dev-server": "^3.7.2"
+  },
+  "resolutions": {
```

Comment:
should we add the overrides alongside resolutions for npm and yarn?

Reply:
We should, but this change was talked about some time ago, it is something that every botbuilder should have because there are only resolutions.

Reply:
fixed in the bot